src/AutorestSwift/Code Generator/Manager.swift

```diff
@@ -157,6 +157,10 @@ class Manager {
                     .warn(
                         "Discrepancies found in round-tripped code model. Run a diff on 'before.json' and 'after.json' to troubleshoot."
                     )
+                SharedLogger
+                    .warn("beforeJsonUrl=\(beforeJsonUrl)")
+                SharedLogger
+                    .warn("afterJsonUrl=\(afterJsonUrl)")
```

Comment:
Was this just present for debugging?

Reply:
I leave that thinking it can be useful for future debugging. But I have reverted this change.

Reply:
It saved the two files to the generated folder, which would be easier to compare since you can diff with FileMerge (or something similar). My concern leaving this in would be just log spam.